tests/suites/test_suite_psa_its.function

```diff
@@ -190,18 +198,22 @@ void get_at( int uid_arg, data_t *data,
     size_t length = length_arg >= 0 ? length_arg : 0;
     unsigned char *trailer;
     size_t i;
+    size_t ret_len = 0;
 
     ASSERT_ALLOC( buffer, length + 16 );
     trailer = buffer + length;
     memset( trailer, '-', 16 );
 
     PSA_ASSERT( psa_its_set_wrap( uid, data->len, data->x, 0 ) );
 
-    status = psa_its_get( uid, offset, length_arg, buffer );
+    status = psa_its_get( uid, offset, length_arg, buffer, &ret_len );
     TEST_ASSERT( status == (psa_status_t) expected_status );
     if( status == PSA_SUCCESS )
+    {
+        TEST_ASSERT( (size_t) length_arg == ret_len );
         ASSERT_COMPARE( data->x + offset, length,
```

Comment:
We can do this in one assert if we replace the second `length` with `length_arg`. Same applies to other similar asserts added in this PR.

Reply:
I think you mean the code changes to the following (which uses ret_len rather than length_arg): 

```
    if( status == PSA_SUCCESS )
        ASSERT_COMPARE( data->x + offset, length,
                        buffer, ret_len );
```

Reply:
Yes, I like your suggestion better, to match what's passed to `psa_its_get()`.

Reply:
Ok, will change it to the new version.